tests/test_agnostic/test_utils.py

```diff
@@ -36,34 +42,34 @@ def test_validate(tmp_path):
         "roi_max": [100, 100],
     }
     # No nodes
-    with pytest.raises(AssertionError, match="graph group must contain a nodes group"):
+    with pytest.raises(ValueError, match="'graph' group must contain a group named 'nodes'"):
         validate(zpath)
-    z.create_group("nodes")
+    nodes = z.create_group("nodes")
 
     # Nodes missing ids
-    with pytest.raises(AssertionError, match="nodes group must contain an ids array"):
+    with pytest.raises(ValueError, match="'nodes' group must contain an 'ids' array"):
```

Comment:
in many of these changes, we're just putting the name of the group in quotes (makes it stand out)